spec/system/visit_index_page_spec.rb

```diff
@@ -27,14 +33,15 @@
     expect(page).to have_content 'Manage certificates'
   end
 
-  it 'shows index page and successfully goes to next page' do   
+  it 'shows index page and successfully goes to next page' do
     msa_component = msa_encryption_certificate.component
     visit root_path
     expect(page).to have_content 'Manage certificates'
-    within("##{msa_component.id}") do 
-      click_link('Encryption certificate')
+    within("##{msa_component.id}") do
+      click_link('Encryption certificate', match: :first)
```

Comment:
why are we adding the `match: :first` to the existing test? Encryption certificate should always be only one within a component so this should not be needed.

Reply:
Sneaked in the process of rebase previous commits